src/pages/authorization/authorizing-api-calls/index.mdx

```diff
@@ -763,7 +763,7 @@ Your web browser will automatically store and attach these cookies to future req
 
 ```shell
 curl "https://accounts.livechat.com/customer/token" \
-  -X POST
+  -X POST \
```

Comment:
Hello @bolchowka :) I would appreciate it if you could just confirm that these are the correct changes connected with the case reported on [platform-docs](https://livechat.slack.com/archives/C2XMTMQ5S/p1658306967536979). I am fixing two issues in this PR, so please do not pay attention to the above change. Thanks :)